outline-minor-faces.el

```diff
@@ -105,7 +105,11 @@ be used directly because it is only supposed to match the
 beginning of a heading.")
 
 (defun outline-minor-faces--syntactic-matcher (regexp)
-  "Returns a matcher that matches REGEXP only if it is not found inside a string. Returns REGEXP directly for modes where `font-lock-keywords-only' is non-nil because Font Lock does not mark strings and comments for those modes, and the matcher will not know what is/is not a string. "
+  "Returns a matcher that matches REGEXP only if it is not found
```

Comment:
The first sentence must completely fit on the first line. Maybe "Return a matcher that matches REGEXP only outside of strings." Generally if such a first (aka summary) sentence is not precise enough, then that information should be repeated and augmented in the text that follows, even if that leads to some duplicated information.

And there's no need to do this in a separate commit, just squash into the appropriate existing commit. Rewriting "history" (aka it's not actually history yet) is perfectly okay on feature branches, even when they are public.